action.yml

```diff
@@ -38,7 +38,7 @@ runs:
   using: 'docker'
   # This is the image that will be used to run the action. 
   # IMPORTANT: The version has to be changed manually in your PRs.
-  image: 'docker://asyncapi/github-action-for-cli:3.1.1'
+  image: 'docker://asyncapi/github-action-for-cli:3.1.2'
```

Comment:
@derberg could you please check if the bump is correct ( always get confused regarding this 😢 ).